src/index.js

```diff
@@ -1 +1,55 @@
 import './styles.css';
+
+const itemList = document.getElementById('poke-list');
+
+const getPokemonIdFromURL = (url) => {
+  const parts = url.split('/');
+  return parts[parts.length - 2];
+};
+
+fetch('https://pokeapi.co/api/v2/pokemon?offset=3&limit=6')
+  .then((response) => response.json())
+  .then((data) => {
+    const itemGrid = document.createElement('div');
+    itemGrid.classList.add('poke-grid');
+
+    data.results.forEach((pokemon) => {
+      const item = document.createElement('div');
+      item.classList.add('item');
+
+      const title = document.createElement('h2');
+      title.textContent = pokemon.name;
+
+      const image = document.createElement('img');
+      const pokemonId = getPokemonIdFromURL(pokemon.url);
+      image.src = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/${pokemonId}.svg`;
+      image.alt = pokemon.name;
+
+      const buttonContainer = document.createElement('div');
+      buttonContainer.classList.add('button-container');
+
+      const commentsButton = document.createElement('button');
+      commentsButton.classList.add('button');
+      commentsButton.textContent = 'Comments';
+
+      const reservationsButton = document.createElement('button');
```

Comment:
Considering the fact that we're not implementing the `Reservation` feature, do you think it might be a good idea to remove it altogether from out app?